src/hardware/adlib.cpp

```diff
@@ -164,7 +165,48 @@ struct Handler : public Adlib::Handler {
 
 }
 
+namespace NukedOPL {
 
+struct Handler : public Adlib::Handler {
+	opl3_chip chip = {};
+	Bit8u newm = 0;
+
+	void WriteReg(Bit32u reg, Bit8u val) override
+	{
+		OPL3_WriteRegBuffered(&chip, (Bit16u)reg, val);
+		if (reg == 0x105)
+			newm = reg & 0x01;
+	}
+
+	Bit32u WriteAddr(Bit32u port, Bit8u val) override
+	{
+		Bit16u addr;
+		addr = val;
+		if ((port & 2) && (addr == 0x05 || newm)) {
+			addr |= 0x100;
+		}
+		return addr;
+	}
+
+	void Generate(MixerChannel *chan, Bitu samples) override
+	{
+		Bit16s buf[1024 * 2];
+		while (samples > 0) {
+			Bit32u todo = samples > 1024 ? 1024 : samples;
```

Comment:
RAW_SIZE and OPL_WRITEBUF_SIZE are both `#defined to 1024. Are either valid here?

Reply:
OPL_WRITEBUF_SIZE seems internal to nuked, so perhaps better not.

RAW_SIZE is an unused leftover of code, that was removed some time ago: introduced in  dc6e126a719217b14de43c02eb34b11eea97fc6a usage removed in 1ff3aed302c5ad1c555a20e9830c8e28ad749aa0 and then the line was touched in 9b70d0085 (gitk is really useful for such investigations). We could move the define around and repurpose it… But all opl handler implementations use bufs of multiples of 1024… perhaps it's best to leave it as it is.

I will remove unused RAW_SIZE define from the code.